pages/coinbase/signin.py

```diff
@@ -8,12 +8,35 @@
 
 class SigninPage:
 
-    def __init__(self, driver, drivethrough=True, bypass_auth=True):
+    def __init__(self, driver, spawn=False, bypass_auth=True):
+        """
+        inputs
+        -----
+        driver = selenium driver
+        spawn: bool
+            The default use case is that a driver that is already at the 
+            appropriate url is passed in as part of a user workflow. if 
+            spawn=True, the driver will be navigated to the appropriate url on 
+            instantiation.
+        bypass_auth: bool
+            Two-factor authentication is manditory. When bypass_auth is true,
+            cookies for the option to bypass are added before login. The cookie
+            file is in /secure and not tracked, so one must be built:
+            1. Log in manually, choosing the option to skip two-factor auth next
+               time
+            2. Log out
+            3. Do driver.get_cookies and print the results (it's a list)
+            4. Make a file in /secure, declare a single variable, and copy and
+               paste the cookie list into the variable
```

Comment:
Because in the cookie loading code the file and variable is defined, shouldn't you give the explicit name of these in this directions?

Reply:
I should actually take this out because it's in README.me, which is in another PR. I'll just remove this.